operator/src/main/java/org/bf2/operator/operands/KafkaCluster.java

```diff
@@ -475,6 +475,12 @@ private KafkaClusterTemplate buildKafkaTemplate(ManagedKafka managedKafka, int r
 
         podTemplateBuilder.addAllToTolerations(OperandUtils.profileTolerations(managedKafka));
 
+        if (replicas == 1) {
```

Comment:
How about adding a `managedKafka.isDeveloperInstance()` method in `ManagedKafka` method that relies on https://github.com/bf2fc6cc711aee1a0c2a/kas-fleetshard/blob/6aa4ac43d6eec97d2ba403bf8229269c2f5c85db/api/src/main/java/org/bf2/operator/resources/v1alpha1/ManagedKafka.java#L39?

Reply:
Not that we currently have additional profile plans, but the intent of checking the replicas instead of the profile directly was that we don't care about HA when the number of replicas is 1 and thus it's movable.  

As mentioned in the description we may eventually want to also tie this logic to whether bin-packing is being used or not.